cookbooks/transform_fault_behn_2007/doc/temperature.part.prm

```diff
@@ -0,0 +1,28 @@
+# We use the Geodynamic World Builder to create the initial temperature.
```

Comment:
```suggestion
# This cookbook illustrates how to use the Geodynamic World Builder to model the thermal structure of oceanic transform faults , following Behn et al. (2007).
# We use the Geodynamic World Builder to create the initial temperature.
```
Or something similar to the first sentence, I find it helpful to have a brief 1-2 sentence overview at the beginning.

Reply:
This is just the part of the prm file that shows up in the manual section of the cookbook (included through the md file). There is text beforehand that gives an overview. 
The full prm has the text
```
# This is a model of a mid-ocean ridge with a transform fault,
# specifically, it is a model that reproduces the setup of Behn et
# al., 2007: Thermal structure of oceanic transform faults.
# This input file covers case 1 from that publication (with a
# constant viscosity).
```
as an overview.